sdk/openai/Azure.AI.OpenAI/api/Azure.AI.OpenAI.netstandard2.0.cs

```diff
@@ -82,6 +82,8 @@ public partial class EmbeddingsUsage
     public partial class OpenAIClient
     {
         protected OpenAIClient() { }
+        public OpenAIClient(string token) { }
```

Comment:
(Note: this can come later, far from blocking!) along the same lines as "should we make a typed wrapper for this token to make it clear it's not *Azure* OpenAI," should we give the parameter name a bit more detail at this stage?

Using OpenAI's docs as inspiration:
https://platform.openai.com/docs/api-reference/authentication

It could be something like "nonAzureApiKey," "openAiApiKey," "externalApiKey," etc.; I think we for sure want to use "key" rather than "token" in there.

Reply:
In addition to allowing this be rolled, should it also be automatically refreshable?  See https://apiview.dev/Assemblies/Review/6badd6f3e0194ff2a17a3da02ee45dcd#Azure.Communication.CommunicationTokenCredential and https://apiview.dev/Assemblies/Review/6badd6f3e0194ff2a17a3da02ee45dcd#Azure.Communication.CommunicationTokenRefreshOptions for prior art.

Reply:
And maybe `openAIAuthToken` as a stop gap name-only improvement to make it clearer?